integration/js/VideoPinTest.js

```diff
@@ -0,0 +1,32 @@
+const {TestUtils} = require('kite-common');
+const SdkBaseTest = require('./utils/SdkBaseTest');
+const {OpenAppStep, AuthenticateUserStep, ClickStartLocalVideoButton, ClickStopLocalVideoButton, ClickUnbindVideoElementButton, ClickBindVideoElementButton} = require('./steps');
```

Comment:
`
const { OpenAppStep, AuthenticateUserStep, ClickStartLocalVideoButton, ClickStopLocalVideoButton, ClickUnbindVideoElementButton, ClickBindVideoElementButton } = require('./steps');
`

Need spaces { abc }
npm run build:release will prettify the code.

Reply:
Sorry I might miss something here. 
I did run npm run build:release and it did not seem to correct the format or prettify the code under integration?